services/subgraphs.md

```diff
@@ -10,23 +10,16 @@ The Isolated Pools subgraphs provides endpoints for querying data related to Iso
 ### Deployments
 [BNB Chain](https://thegraph.com/explorer/subgraphs/H2a3D64RV4NNxyJqx9jVFQRBpQRzD6zNZjLDotgdCrTC?view=Query&chain=arbitrum-one)
 
-[Chapel](https://testnet.thegraph.com/explorer/subgraphs/mkXvkAme9PyDnSrLkajVHKVX7eG2HHATcYPE1qGF7gc?view=Query&chain=arbitrum-sepolia)
-
 [Ethereum](https://thegraph.com/explorer/subgraphs/Htf6Hh1qgkvxQxqbcv4Jp5AatsaiY5dNLVcySkpCaxQ8?view=Query&chain=arbitrum-one)
 
-[Sepolia](https://testnet.thegraph.com/explorer/subgraphs/6YURB3bABVUNusgjtY7fNMkELGmKm9aaWirMQ9UZeDFj?view=Query&chain=arbitrum-sepolia)
-
 [OpBnB](https://open-platform-ap.nodereal.io/{apikey}/opbnb-mainnet-graph-query/subgraphs/name/venusprotocol/venus-isolated-pools-opbnb/graphql)
```

Comment:
We need to update the apikey within this URL, otherwise as a link from the documentation it doesn't work.